mypyc/test-data/run-tuples.test

```diff
@@ -95,6 +95,73 @@ class Sub(NT):
     pass
 assert f(Sub(3, 2)) == 3
 
+-- Ref: https://github.com/mypyc/mypyc/issues/924
+[case testNamedTupleClassSyntax]
+from typing import Dict, List, NamedTuple, Optional, Tuple, Union
+
+class FileData(NamedTuple):
+    st_mtime: float
+    st_size: int
+    hash: str
+
+class SearchPaths(NamedTuple):
+    python_path: Tuple[str, ...]
+    mypy_path: Tuple[str, ...]
+    package_path: Tuple[str, ...]
+    typeshed_path: Tuple[str, ...]
+
+class ClassIR: pass
+
+class FuncIR: pass
+
+class VTableMethod(NamedTuple):
+    cls: 'ClassIR'
+    name: str
+    method: FuncIR
+    shadow_method: Optional[FuncIR]
+
+class DeserMaps(NamedTuple):
+    classes: Dict[str, 'ClassIR']
+    functions: Dict[str, 'FuncIR']
```

Comment:
These two fields are functionally the same, right? In general this test feels a bit too verbose, maybe remove some fields that don't test anything new.